scripts/sync-schema-local.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+
+cd ../metaphysics
+yarn dump-schema v2 "$OLDPWD/data/"
```

Comment:
@pvinis - these are nice additions, but wondering why not just use the command in `package.json`?

Reply:
We used to have these in `package.json`, but because force uses node 14 and mp node 10 or something like that, I would get errors because of version mismatch. With an actual bash script, after `cd` the node version is the correct local one for each dir. In yarn scripts that doesn't happen. Does that make sense?